.markdown-doctest-setup.js

```diff
@@ -0,0 +1,53 @@
+function noop () {}
+
+function jQuery () {
+  return {
+    on: function (ev, cb) {
+      cb()
+    }
+  }
+}
+
+jQuery.each = function (items, f) {
+  items.forEach(f)
+}
+
+module.exports = {
+  globals: {
+    '_': require(__dirname),
+
+    asyncSave: noop,
+    addContactToList: noop,
+    calculateLayout: noop,
+    createApplication: noop,
+    updatePosition: noop,
+    sendMail: noop,
+    renewToken: noop,
+    batchLog: noop,
+
+    setImmediate: setImmediate,
+    Buffer: Buffer,
+    EventSource: function () {},
+
+    fs: {writeFileSync: noop},
+    path: {join: noop},
+
+    cwd: __dirname,
+    mainText: '',
+    data: {user: 'Quinoa'},
```

Comment:
It's used in this example:

```js
// Use the `variable` option to ensure a with-statement isn't used in the compiled template.
var compiled = _.template('hi <%= data.user %>!', { 'variable': 'data' });
compiled.source;
// => function(data) {
//   var __t, __p = '';
//   __p += 'hi ' + ((__t = ( data.user )) == null ? '' : __t) + '!';
//   return __p;
// }
```

Quinoa is a bit of a running joke between myself and some of my coworkers. Usually when programmers pick names for examples, they pick something like 'Steve' or 'Mike'. We ended up picking 'Quinoa' as a name of one of our test customers, and it's been my go to ever since. I can change it to something more reasonable if you like!